tests/test_bert_classifier.py

```diff
@@ -1,12 +1,21 @@
 # encoding: utf-8
+import pytest
 import tempfile
 
 import numpy as np
 
 from wellcomeml.ml.bert_classifier import BertClassifier
 
 
-def test_multilabel():
+@pytest.fixture
+def multilabel_bert():
```

Comment:
we can use scope module here to invoke once https://docs.pytest.org/en/latest/how-to/fixtures.html#fixture-scopes

Reply:
Good shout, forgot about the scope!

Reply:
https://github.com/wellcometrust/WellcomeML/pull/273/commits/8a9b45a47d03a244e8280180bd4cbb2a4fb06f34